.github/workflows/test.yml

```diff
@@ -3,13 +3,23 @@ name: pymongo-auth-aws
 on: [push, pull_request]
 
 jobs:
-  build:
 
+  static:
+    runs-on: ubuntu-latest
+    steps:
+     - uses: actions/checkout@v2
+     - uses: actions/setup-python@v3
+     - uses: pre-commit/action@v3.0.1
```

Comment:
Are we able to refer to any repo in `https://github.com/actions` without any further setup? Does the repo just need an `action.yml`?

Reply:
> Does the repo just need an action.yml?

Yep